src/open/clients/Client.js

```diff
@@ -0,0 +1,35 @@
+import { Platform } from "../types.js";
+
+export class Client {
+    constructor(data) {
+        this.data = data;
+    }
+
+    get id() { return this.data.id; }
+
+    get platforms() { return this.data.platforms; }
+
+    get icon() { return "images/client-icons/"+this.data.icon; }
```

Comment:
Is it safe to assume client icons are always going to be in images/client-icons? Are there reasons for it not to be the case?

Reply:
yeah, seems ok to me